### PR TITLE
Event search

### PR DESCRIPTION
**Pull request by sirlensalot**

### CODE REVIEW

**Review by mightybyte — CHANGES_REQUESTED** [review]

File: frontend/src/Frontend/Transactions.hs [hunk 1]
```diff
@@ -160,13 +219,39 @@ txTable net txs = do
...
         elAttr "td" ("data-label" =: "Chain") $ text $ tshow chain
         elAttr "td" ("data-label" =: "Height") $ blockLink net (ChainId chain) height $ tshow height
-        elAttr "td" ("data-label" =: "Sender") $ senderWidget tx
-        elAttr "td" ("data-label" =: "Request Key") $ do
-          let contents = case (_txSummary_code tx, _txSummary_continuation tx) of
-                           (Just c, _) -> _txSummary_requestKey tx
-                           (_, Just v) -> showCont v
-                           (_, _) -> ""
-          text contents
+        elAttr "td" ("data-label" =: "Request Key") $ txDetailLink net (_txSummary_requestKey tx) (_txSummary_requestKey tx)
```
[the remark this answers is not in the document]
Comment (sirlensalot) [reply frontend/src/Frontend/Transactions.hs:222]: I'm going to work on a results summary with elided code etc

File: frontend/src/Frontend/Transactions.hs [hunk 2]
```diff
@@ -138,35 +190,91 @@ txTable
...
+        elAttr "td" ("class" =: "center aligned" <> "data-label" =: "Status")
+            status
+        elAttr "td" ("data-label" =: "Chain") $
+            text $ tshow chain
+        elAttr "td" ("data-label" =: "Time") $
+            text $ T.take 19 $ tshow (_txSummary_creationTime tx)
+        elAttr "td" ("data-label" =: "Height") $
+            blockLink net (ChainId chain) height $ tshow height
+        elAttr "td" ("data-label" =: "Request Key") $
+            txDetailLink net rk (T.take 24 rk <> "...")
```
Comment (mightybyte) [inline frontend/src/Frontend/Transactions.hs:227]: I'd probably move the manual ellipsis into CSS with something like this:

```
  overflow: hidden;
  text-overflow: ellipsis;
```

...or similar.  But we can do that later.

Comment (mightybyte) [reply frontend/src/Frontend/Transactions.hs:227]: The advantage of doing it that way is that the user can still copy/paste the full value even if it doesn't fit.  See, for example https://github.com/kadena-io/chainweaver/blob/3ea0bb317c07ddf954d4ebf24b33d1be7d5f9c45/backend/sass/common.blocks/_wallet.scss#L47

**Review by mightybyte — APPROVED** [review]

File: frontend/src/Frontend/Transactions.hs (hunk 2 above)
Comment (sirlensalot) [reply frontend/src/Frontend/Transactions.hs:227]: Yeah that's good stuff. We need it for bond explorer eliding too.

**Pull request merged**